core/src/main/java/io/confluent/kafka/schemaregistry/rest/SchemaRegistryConfig.java

```diff
@@ -314,6 +316,9 @@
       + "`ssl.truststore.` configs are used while making the call. The "
       + "schema.registry.inter.instance.protocol name is deprecated; prefer using "
       + "inter.instance.protocol instead.";
+  private static final String SCHEMAREGISTRY_HEADERS_FORWARD_DOC
+      = "A list of `http` headers to forward from slave to master, "
```

Comment:
If the purpose of the backticks is to cause the resulting RST doc to render `like this`, they should be doubled: ` ``http`` ` instead of `` `http` ``, ` ``Content-Type`` ` instead of `` `Content-Type` ``, etc.